stan/model.py

```diff
@@ -87,6 +87,26 @@ def hmc_nuts_diag_e_adapt(self, **kwargs) -> stan.fit.Fit:
         kwargs["function"] = "stan::services::sample::hmc_nuts_diag_e_adapt"
         return self._create_fit(kwargs)
 
+    def fixed_param(self, **kwargs) -> stan.fit.Fit:
+        """Draw samples from the model using ``stan::services::hmc_nuts_diag_e_adapt``.
```

Comment:
```suggestion
        """Draw samples from the model using ``stan::services::fixed_param``.
```

Is it `stan::services::sample::fixed_param` or `stan::services::fixed_param`

Reply:
Thanks. You're right about this. It is ``stan::services::sample::fixed_param``. We should fix this for everything.